### DISCUSSION TITLE
分支名起错了怎么办?

### DISCUSSION THREAD

**Discussion by labike** [greyli/flask-docs-zh#62] [Q&A] [answered]
当时准备翻译async/await呢，建了个async/await的分支，后来你告诉我这个有人领 了, 我就去翻译views了, 结果现在在async/await的分支中翻译了views, 这个咋弄呀？

**Comment by labike** [comment]
![views](https://user-images.githubusercontent.com/15871933/120597904-350b8880-c478-11eb-9394-7040d6dfa595.png)
![async](https://user-images.githubusercontent.com/15871933/120597925-3b016980-c478-11eb-8894-e08a6608c09e.png)

**Comment by rosekc** [comment]
大概没事。分支改名可以这样：

```sh
git branch -m new_branch_name
```

github 仓库里的分支可以先删了再推。

参见 https://stackoverflow.com/questions/9524933/renaming-a-branch-in-github

**Comment by labike** [comment]
![1622708533(1)](https://user-images.githubusercontent.com/15871933/120612728-12816b80-c488-11eb-8a13-256be6e8bc8d.png)
怕提交pr跟别人重了

**Comment by rosekc** [comment, marked as answer]
> ![1622708533(1)](https://user-images.githubusercontent.com/15871933/120612728-12816b80-c488-11eb-8a13-256be6e8bc8d.png)
> 怕提交pr跟别人重了

这是自己的仓库，不会重的。如果不是强迫症不改大概也无伤大雅。
